### PR TITLE
Keck hires dev

### PR DESCRIPTION
**Pull request by jhennawi**
Updates the dev suite to include ongoing HIRES development.

### CODE REVIEW

File: unit_tests/test_setups.py [hunk 1]
```diff
@@ -281,6 +281,99 @@ def test_setup_keck_deimos_multiconfig_clean():
     assert len(ps.fitstbl) == 23, 'Incorrect number of table rows.'
 
 
+def test_setup_keck_mosfire():
```
Comment (profxj) [inline unit_tests/test_setups.py:284]: is this really part of this PR?

**Review by profxj — CHANGES_REQUESTED** [review]
I think you meant to add scripts "testing" the new HIRES code.

File: unit_tests/test_setups.py (hunk 1 above)
Comment (jhennawi) [reply unit_tests/test_setups.py:284]: I definitely did not write this, but perhaps something did not get merged into develop when I pulled it in.

**Comment by jhennawi** [comment]
The tests of the code are in PypeIt when you run run_pypeit on keck_hires data in the dev suite. I don't plan to write any others tests as the code is not finished yet. I've checked in the place where I've developed, but there may be some overlap with what is in PypeIt. Anyway, the official version is in PypeIt.

**Comment by profxj** [comment]
If that means there is a PypeIt file (it must),
them please add that in.

**Comment by kbwestfall** [comment]
Already merged as part of the JWST+HIRES PR.

**Pull request closed without merging**